spec/fixtures/teachers.yml

```diff
@@ -8,8 +8,8 @@ ye:
   email: 'ye@berkeley.edu'
```

Comment:
What is motivating your use of fixtures here? Was this by Michael's guidance?

In the PR description, you mention "fixtures/email_templates.yml: the problem here is i can't seem to put erb things. So might be non-trivial to set up comprehensive tests" -- were you trying to use erb-like syntax to dynamically set values and vary the data to be tested? Fixtures wouldn't be a good match then

Reply:
Discussed in person: fixtures were already used in the legacy app, so they were extended upon. Recommendation given to switch to something like factories for best practices, but not a top priority for now.